info-leak.py

```
@@ -0,0 +1,203 @@
+            with open(file_chooser.getSelectedFile().getAbsolutePath(), "r") as f:
+                grep_strings = json.load(f)
+            self._grep_strings_area.setText("\n".join(grep_strings))
+
+    def _export_grep_strings(self, event):
+        file_chooser = JFileChooser()
+        if file_chooser.showSaveDialog(None) == JFileChooser.APPROVE_OPTION:
+            with open(file_chooser.getSelectedFile().getAbsolutePath(), "w") as f:
+                grep_strings = self._grep_strings_area.getText().split("\n")
+                json.dump(grep_strings, f)
+
+    def _save_grep_strings(self, event):
```

Ruff (0.12.2)

Unused method argument: `event`

(ARG002)